### ISSUE TITLE
Change the order of indexing in the generated batch

### ISSUE THREAD

**Issue by atorchylo** [LSSTDESC/BlendingToolKit#346]
Currently it is `batch[key][survey_name]`. I think, it makes more sense to have `batch[survey_name][key]`

<img width="1256" alt="Screen Shot 2022-08-05 at 12 43 02 PM" src="https://user-images.githubusercontent.com/61771552/183150137-b957b243-d744-4f85-8f32-8247d19437d5.png">

**Comment by aboucaud** [comment]
Indeed I feel the order you are suggesting is more natural.
But if you make that switch, you have to keep in mind using this generator with a single survey will end up being more cumbersome
E.g. for a generator with LSST only:
```python
batch['LSST']['blend_images']
batch['LSST']['isolated_images']
batch['LSST']['blend_list']
```

**Comment by ismael-mendoza** [comment]
resolved in #412
